<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1219</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineE_Num">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Sp">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Simple</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Double</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushB_Cal">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Original Number:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineE_Onum">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>551</width>
      <height>28</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>109</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Fixed point number:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineE_Pf">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>551</width>
      <height>28</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>140</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Fixed point number (Normalized):</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineE_PfN">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>150</y>
      <width>551</width>
      <height>28</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Exponent calculation:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineE_Ce">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>190</y>
      <width>551</width>
      <height>28</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculation of the mantissa:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineE_Cm">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>230</y>
      <width>551</width>
      <height>28</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Format</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>270</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>S</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineE_fE">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Exponent</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineE_fF">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>270</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Mantissa</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_9">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Binary</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_10">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>360</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Hexa</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_11">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>470</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Binary</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_12">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>500</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Hexa</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_13">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>410</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Format</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_14">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>410</y>
      <width>641</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Mantissa</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Precision:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>330</y>
      <width>641</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>470</y>
      <width>641</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2"/>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>300</y>
      <width>641</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridL_num1"/>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>440</y>
      <width>641</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridL_num2"/>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>360</y>
      <width>641</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3"/>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>500</y>
      <width>641</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4"/>
   </widget>
   <widget class="QPlainTextEdit" name="plainTE_Desc">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>70</y>
      <width>391</width>
      <height>461</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>30</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Process description</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>lineE_fF</zorder>
   <zorder>lineE_Num</zorder>
   <zorder>comboBox_Sp</zorder>
   <zorder>pushB_Cal</zorder>
   <zorder>label</zorder>
   <zorder>lineE_Onum</zorder>
   <zorder>label_2</zorder>
   <zorder>lineE_Pf</zorder>
   <zorder>label_3</zorder>
   <zorder>lineE_PfN</zorder>
   <zorder>label_4</zorder>
   <zorder>lineE_Ce</zorder>
   <zorder>label_5</zorder>
   <zorder>lineE_Cm</zorder>
   <zorder>lineEdit_5</zorder>
   <zorder>lineEdit_6</zorder>
   <zorder>lineE_fE</zorder>
   <zorder>lineEdit_9</zorder>
   <zorder>lineEdit_10</zorder>
   <zorder>lineEdit_11</zorder>
   <zorder>lineEdit_12</zorder>
   <zorder>lineEdit_13</zorder>
   <zorder>lineEdit_14</zorder>
   <zorder>label_6</zorder>
   <zorder>gridLayoutWidget</zorder>
   <zorder>gridLayoutWidget_2</zorder>
   <zorder>gridLayoutWidget_3</zorder>
   <zorder>gridLayoutWidget_4</zorder>
   <zorder>gridLayoutWidget_5</zorder>
   <zorder>gridLayoutWidget_6</zorder>
   <zorder>plainTE_Desc</zorder>
   <zorder>label_7</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
